server/chart_config/climate.json

```diff
@@ -10,5 +10,19 @@
       "scale": false
     },
     "isChoropleth": true
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-Daily_max_temperature_rcp85_diff_to_2006",
+    "title": "Daily max temperature change (from year 2006)",
```

Comment:
we should add that it's under RCP 8.5 scenario. maybe:

Projected max daily temperature change under RCP 8.5 (based on year 2006)

Reply:
Done